test/test_cosmic_mutations.py

```diff
@@ -9,11 +10,11 @@
     FrameShiftTruncation,
 )
 
-annot = VariantAnnotator(75)
+ensembl = EnsemblRelease(75)
```

Comment:
Can you say more about why that's a better name?
On Feb 20, 2015 11:35 AM, "timodonnell" <notifications@github.com> wrote:

> In test/test_cosmic_mutations.py
> <https://github.com/hammerlab/varcode/pull/20#discussion_r25082212>:
>
> > @@ -9,11 +10,11 @@
> >      FrameShiftTruncation,
> >  )
> >
> > -annot = VariantAnnotator(75)
> > +ensembl = EnsemblRelease(75)
>
> I'd suggest renaming this default_ensembl
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/varcode/pull/20/files#r25082212>.
>